zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -397,7 +372,7 @@ benchmarks! {
         let d in 0..T::MaxDisputes::get();
 
         let c_u16 = c.saturated_into();
-        let (caller, market_id) = setup_resolve_common_categorical_after_dispute::<T>(a, b, c_u16)?;
+        let (caller, market_id) = setup_resolve_common_categorical_after_dispute::<T>(b, c_u16)?;
```

Comment:
I would expect to benchmark `PedictionMarket::on_resolution` here (not `SimpleDisputes::on_resolution`).